pkg/types/notification.go

```diff
@@ -35,6 +35,9 @@ const (
 
 	// DELETED represents a notification type for deleting a resource
 	DELETED OperationType = "DELETED"
+
+	// INVALIDREVISION revision with invalid value
+	INVALIDREVISION int64 = -1
```

Comment:
It is used in the storage/postgres and api/notifications. It will be strange for any of them to import the other. It cannot be in storage as well as postgres must not import storage. This seemed the most appropriate as InvalidRevision is some value of a property of notification similar to the operation types (CREATED, DELETED ...).

Reply:
Can it be `InvalidNotificationRevision` then, because `types.InvalidRevision` doesn't mean much